projects/extension/package.json

```diff
@@ -32,6 +32,7 @@
   "dependencies": {
     "@substrate/connect-extension-protocol": "^1.0.1",
     "@substrate/smoldot-light": "0.7.2",
+    "multiaddr": "^10.0.1",
```

Comment:
I'm vetoing this dependency. The set of multiaddresses that we support is way smaller than the set of multiaddresses that this library is capable of parsing.

Please just check whether the multiaddress matches one of these two regexes:

```js
/^\/(ip4|ip6|dns4|dns6|dns)\/(.*?)\/tcp\/(.*?)\/(ws|wss|tls\/ws)\/p2p\/(.*?)$/
/^\/(ip4|ip6)\/(.*?)\/udp\/(.*?)\/webrtc\/certhash\/(.*?)\/p2p\/(.*?)$/
```

Then verify that the thing after `/p2p` is valid base58, and that the thing after `/certhash` (in the second regex) is valid base64.